ui/src/testing/enzyme-intl-helper.js

```diff
@@ -69,3 +69,7 @@ export function mountWithIntl(node, { context, childContextTypes, ...additionalO
         }
     );
 }
+
+export async function asyncMountWithIntl(node, { context, childContextTypes, ...additionalOptions } = {}) {
```

Comment:
I'm surprised calling `await mountWithIntl` didn't work. Maybe chuck a comment on here for future explorers.